CONTRIBUTING.md

```diff
@@ -330,6 +331,22 @@ internals::impl_from_infallible!(Error);
 All errors that live in an `error` module (eg, `foo/error.rs`) and appear in a public function in
 `foo` module should be available from `foo` i.e., should be re-exported from `foo/mod.rs`.
 
+##### `expect` messages
+
+With respect to `expect` messages, they should follow the
+[Rust standard library guidelines](https://doc.rust-lang.org/std/option/enum.Option.html#recommended-message-style).
+More specifically, `expect` messages should be used to to describe the reason
+you expect the operation to succeed.
+For example, this `expect` message clearly states why the operation should succeed:
+
+```rust
+let item = slice.get(0)
+    .expect("slice should not be empty");
```

Comment:
That's also not great message. It generally should describe why the condition holds. E.g. "checked above" or "empty slice disallowed by function's contract". "should not be" is too vague.

Reply:
That's a direct copy and paste from https://doc.rust-lang.org/std/option/enum.Option.html#recommended-message-style

Reply:
LOL, so even official doc gets it wrong. I might send them a PR tomorrow. :)

Reply:
What would you suggest here, @Kixunil?

Reply:
A more realistic example.

```rust
match slice.len() {
    21 => fn_accepting_arrays_with_len_21(slice.try_into().expect("statically known length")),
    42 => fn_accepting_arrays_with_len_42(slice.try_into().expect("statically known length")),
    _ => return Err(InvalidSliceLen)
}
```

You should be able to find something very similar to this somewhere (I think in `key.rs`?), so you might as well copy that.

Reply:
Thanks I've found one in `key.rs`